src/components/posts/PostLike.js

```diff
@@ -0,0 +1,66 @@
+import like from './like.png';
+import React from 'react';
+import { useState, useEffect } from 'react';
+import client from '../../utils/client';
+import './style.css';
+
+const blue = '#1976d2';
+
+const PostLike = ({likes=[], postId}) => { 
+const [divStyle, updateStyle] = useState({
+  backgroundColor: 'grey'
+  });
+const [postLikes,updateLikes] = useState(likes);
+const [id, updateId] = useState('');
+const [talkingToServer, updateTalkStatus] = useState(false);
+
+useEffect(()=>{
+  const like = likes.find(like => String(like.userId) === localStorage.getItem('userId')); // checks if user has liked the post
+  if (like){
+    updateStyle({backgroundColor: blue});
+    updateId(like.id);
+  }
+},[]);
+
+  function likePost(event){
+  updateTalkStatus(true);
+  client.post(`/post/${postId}/like/`)
+  .then(res => {
+    updateId(_ => res.data.data.id);
+    updateLikes(likes => [res.data.data, ...likes]);
+    updateStyle({backgroundColor: blue});
+  })
+  .catch((err) => {
+    console.log({error: err.message});
```

Comment:
Might be an idea to add a little context to this error message. 

Something like "error liking post". 

The reason being is that, when looking at the console, it might not be immediately clear _where_ the error is occurring.